src/corehost/cli/hostpolicy.cpp

```diff
@@ -147,28 +148,31 @@ int run(const arguments_t& args)
         return StatusCode::CoreClrInitFailure;
     }
 
+    // Initialize clr strings for arguments
+    std::vector<std::vector<char>> argv_strs(args.app_argc);
+    std::vector<const char*> argv(args.app_argc);
+    for (int i = 0; i < args.app_argc; i++)
+    {
+        pal::to_clrstring(args.app_argv[i], &argv_strs[i]);
+        argv[i] = argv_strs[i].data();
+    }
+
     if (trace::is_enabled())
     {
         pal::string_t arg_str;
-        for (int i = 0; i < args.app_argc; i++)
+        for (int i = 0; i < argv.size(); i++)
         {
-            arg_str.append(args.app_argv[i]);
+            pal::string_t cur;
+            pal::clr_palstring(argv[i], &cur);
+            arg_str.append(cur);
             arg_str.append(_X(","));
         }
-        trace::info(_X("Launch host: %s app: %s, argc: %d args: %s"), args.own_path.c_str(),
+        trace::info(_X("Launch, host: %s app: %s, argc: %d args: %s"), args.own_path.c_str(),
```

Comment:
No it was added while debugging to detect change. Intent was not to commit it. Thanks.